mycroft/skills/wolfram_alpha/__init__.py

```diff
@@ -123,9 +123,13 @@ def get_result(self, res):
                         result = self.__find_value(
                             res.pods, 'BasicInformation:PeopleData')
                         if not result:
-                            result = self.__find_value(
-                                res.pods, 'DecimalApproximation')
-                            result = result[:5]
+                            result = self.__find_value(res.pods, 'Definition')
+                            if not result:
+                                result = self.__find_value(
+                                    res.pods, 'DecimalApproximation')
+                                if not result:
+                                    result = self.__find_num(
+                                        res.pods, '200')
```

Comment:
This searches for the pod with the `position` attribute 200, which corresponds to the second pod in the XML result. This means that Mycroft will return a result much more frequently rather than saying `Sorry, I don't understand your request` as often.